docker/docker/src/main/java/org/arquillian/cube/docker/impl/docker/DockerClientExecutor.java

```diff
@@ -554,10 +556,15 @@ public void pullImage(String imageName) {
 
     public String execStart(String containerId, String... commands) {
         ExecCreateCmdResponse execCreateCmdResponse = this.dockerClient.execCreateCmd(containerId)
-                .withAttachStdout(true).withAttachStdin(false).withAttachStderr(false).withTty().withCmd(commands)
-                .exec();
-        InputStream consoleOutputStream = dockerClient.execStartCmd(execCreateCmdResponse.getId()).withDetach(false)
+                .withAttachStdout(true).withAttachStdin(false).withAttachStderr(false).withTty(false).withCmd(commands)
                 .exec();
+        try {
+            dockerClient.execStartCmd(execCreateCmdResponse.getId()).withDetach(false)
+                    .exec(new ExecStartResultCallback(System.out, System.err)).awaitCompletion();
+        } catch (InterruptedException e) {
+            return "";
+        }
+        InputStream consoleOutputStream = dockerClient.copyArchiveFromContainerCmd(containerId, "/execStartTest.log").exec();
```

Comment:
This API breaks the backcompatibility from major version to major version which is not good at all for we (the integrators). Have you tried to attach the stdin? https://github.com/docker-java/docker-java/blob/9ec46f44374c434a9b6132cd8611e2c8e80b8c69/src/test/java/com/github/dockerjava/netty/exec/ExecStartCmdExecTest.java#L102